recipe/requirements.txt

```diff
@@ -0,0 +1,2 @@
+codecov
+nose_warnings_filters
```

Comment:
Ooooh... `nose_warning_filters` is a deep rabbit hole indeed, as it uses the python3-only `flit` for building. Not only that, one of its dependencies, `requests_download` has a build dependency on... `flit`!